docs/current_docs/integrations/snippets/gitlab-kubernetes.yml

```diff
@@ -0,0 +1,19 @@
+.dagger:
+  image: alpine:latest
+  variables:
+    # assumes the Dagger Cloud token is
+    # in a masked/protected variable named DAGGER_CLOUD_TOKEN
+    # set via the GitLab UI
+    DAGGER_CLOUD_TOKEN: $DAGGER_CLOUD_TOKEN
+  before_script:
+    - apk add curl
+    - curl -L https://dl.dagger.io/dagger/install.sh | BIN_DIR=/tmp sh
```

Comment:
Using an explicit `DAGGER_VERSION` variable here which matches the Engine version is highly encouraged. If these two diverge, it may lead to incompatibility issues between the CLI & the Engine.

`/tmp` is usually not executable, so installing the `dagger` CLI there may result in this not working, even if `/tmp` is added to `$PATH`. Regardless where `dagger` CLI gets installed, this path should either:
1. Be added to the `$PATH` explicitly
2. Invoked via an absolute path, e.g. `~/.local/bin/dagger` or `~/bin/dagger`

Reply:
- Will `~/.local` and `~/.bin` exist?
- If we change the path here, should we also change it in the `$HOME` variable in the runner config?

Reply:
We don't know wich image with wich operating system is used. Therefore we have to take a common folder, like `/tmp`.
I would also set it in the runner config. When I tested it I found following solution:
```
[[runners]]
  environment = ["HOME=/tmp", ...
  pre_build_script = "export PATH=\"/tmp/:$PATH\""
```
Setting it inside `environment` does not work. The PATH is not available at this time.

Reply:
Done!